DanXiUI/Utils/CustomMarkdown.swift

```diff
@@ -68,18 +68,22 @@ struct CustomImageProvider: ImageProvider {
 }
 
 extension String {
-    /// Convert Treehole-formatted content to basic markdown, stripping images and LaTeX.
+    /// Convert Treehole-formatted content to basic markdown, stripping images, stickers and LaTeX.
     func stripToBasicMarkdown() -> String {
         let text = NSMutableString(string: self)
         
         _ = try? NSRegularExpression(pattern: #"\${1,2}.*?\${1,2}"#, options: .dotMatchesLineSeparators).replaceMatches(in: text, range: NSRange(location: 0, length: text.length), withTemplate: String(localized: "formula_tag", bundle: .module))
+        // Replace Stickers (e.g., ![](dx_cate))
+        let stickerPatterns = Sticker.allCases.map { $0.rawValue }.joined(separator: "|")
```

Comment:
Since we have enumerated all possible stickers, is it necessary to use regular expressions?

Reply:
Performance may be improved by avoid using regular expressions when not necessary.